src/types/pipeline-builds.ts

```diff
@@ -0,0 +1,3 @@
+import { components } from "./core";
+
+export type PipelineBuildPage = components["schemas"]["PipelineBuildResponse"];
```

Comment:
The type `PipelineBuildResponse` isn't a page I think (it would be `Page_PipelineBuildResponse_` then), it's just an option t will create confusion around this type.hat